### DISCUSSION TITLE
Unexpected feature behavior in Google Chrome

### DISCUSSION THREAD

**Discussion by ZigglerPL** [community/community#63018] [Programming Help]
### Select Topic Area

Bug

### Body

Hello, has anyone encountered a problem where the js functions stopped working in the browser? It happened to me that I finished writing the code and the next day one function based on the new Date() object stopped working completely, and it is called correctly in the right context. And the second one based on the setTimeout function, which is designed to remove an element from the DOM tree after 7h removes the element immediately after running the script in the chrome browser. I have everything logically written syntactically, there are no errors in the browser console either. I always make copies of the entire code to be able to go back to the previous code construction. In these copies, these functions have always worked properly, but now even they do not work as they should. The site is slow on my local host, I am not getting new browser updates because I have Windows 7. If the previous working versions of the code are now not working, it can't be the fault of the code itself, what could have happened?

**Comment by Alokray007** [comment]
Hi @ZigglerPL.  I've never encountered this issue earlier but from my knowledge this might be happening due to windows support for the browser as you are running on old windows version. Try updating to newer version of windows or browser and then check. It should work fine.

**Reply by ZigglerPL** [reply]
The only weird thing is that the browser worked fine for me on this system all the time. If it's not updating, then why did something suddenly happen?

**Reply by Alokray007** [reply]
You might have changed the settings mistakely in developer mode or things like that. Try uninstalling the browser then again install and check.

**Reply by ZigglerPL** [reply]
I probably found the cause after several days of struggling to find the problem. In the browser console I found "Invalid Date {}". Thanks for your willingness to help, I was sure it couldn't be the code's fault. And I mistakenly thought it was the browser's fault.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ZigglerPL ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
